komodo/release_transpiler.py

```diff
@@ -2,31 +2,14 @@
 
 import argparse
 import os
-from typing import Dict, Sequence, Union
+from typing import Dict, Union
 
 import yaml
 
 from komodo.matrix import format_release, get_matrix
 from komodo.prettier import load_yaml, write_to_file
 
 
-def get_py_coords(release_base: str, release_folder: str) -> Sequence[str]:
```

Comment:
The commit message is hiding the fact that you are actually removing functionality (but the functionality seems unused, so it is warranted)